Tasks/MSBuild/MSBuild.ps1

```diff
@@ -73,18 +77,57 @@ if ($configuration)
 
 Write-Verbose "args = $args"
 
-if(!$msBuildLocation)
+# Default the msbuildLocationMethod if not specified. The input msbuildLocationMethod
+# was added to the definition after the input msbuildLocation.
+if ($msbuildLocationMethod -ne 'location' -and $msbuildLocationMethod -ne 'version')
```

Comment:
This fixes an existing bug: If the user chooses the radio option "Specify Location", enters a location, and then changes their mind and chooses the radio option "Version", then the PowerShell script will erroneously use the specified location even though the "Version" radio option is selected.